docs/src/SUMMARY.md

```diff
@@ -17,6 +17,7 @@
   - [Where to start on real code](./tutorial-real-code.md)
 
 - [Kani developer documentation](./dev-documentation.md)
+  - [Working with RUSTC](./rustc-hacks.md)
```

Comment:
Nit: Change occurrences of "RUSTC" to \`rustc\`

Reply:
Ok